### PR TITLE
Update plugin creation for 4.3

### PR DESCRIPTION
**Pull request by markstory**
These change would go into 2.next branch to be release alongside CakePHP 4.3

### CODE REVIEW

File: templates/bake/Plugin/tests/bootstrap.php.twig [hunk 1]
```diff
@@ -55,3 +55,18 @@ if (file_exists($root . '/config/bootstrap.php')) {
...
+
+/**
+ * Load schema from a SQL dump file.
+ *
+ * If your plugin does not use database fixtures you can
+ * safely delete this.
+ *
+ * If you want to support multiple databases, consider
+ * using migrations to provide schema for your plugin,
+ * and using \Migrations\TestSuite\Migrator to load schema.
```
Comment (markstory) [inline templates/bake/Plugin/tests/bootstrap.php.twig:67]: I contemplated adding an option to include database fixture setup which would include:

* Adding migrations as dev-dep
* Using the migrator instead.

If folks prefer this approach I can do that as a follow up.

File: tests/comparisons/Plugin/SimpleExample/tests/bootstrap.php [hunk 2]
```diff
@@ -38,3 +38,18 @@
...
+ * safely delete this.
+ *
+ * If you want to support multiple databases, consider
+ * using migrations to provide schema for your plugin,
+ * and using \Migrations\TestSuite\Migrator to load schema.
+ */
+use Cake\TestSuite\Schema\SchemaManager;
+
+// Load a schema dump file.
+SchemaManager::create('test', './tests/schema.sql');
```
Comment (othercorey) [inline tests/comparisons/Plugin/SimpleExample/tests/bootstrap.php:55]:
```suggestion
SchemaManager::create('test', 'tests/schema.sql');
```
Is the `./` required for any sort of security? I think we should just use relative paths from base like most places.

File: tests/comparisons/Plugin/Company/Example/tests/bootstrap.php [hunk 3]
```diff
@@ -38,3 +38,18 @@
...
+ * safely delete this.
+ *
+ * If you want to support multiple databases, consider
+ * using migrations to provide schema for your plugin,
+ * and using \Migrations\TestSuite\Migrator to load schema.
+ */
+use Cake\TestSuite\Schema\SchemaManager;
+
+// Load a schema dump file.
+SchemaManager::create('test', './tests/schema.sql');
```
Comment (othercorey) [inline tests/comparisons/Plugin/Company/Example/tests/bootstrap.php:55]:
```suggestion
SchemaManager::create('test', 'tests/schema.sql');
```

File: templates/bake/Plugin/tests/bootstrap.php.twig (hunk 1 above)
Comment (othercorey) [reply templates/bake/Plugin/tests/bootstrap.php.twig:67]: I'm not so sure about the name `Migrator` after seeing it in docs.

Comment (markstory) [reply templates/bake/Plugin/tests/bootstrap.php.twig:67]: We can still rename it if you have a more appropriate name in mind.

File: templates/bake/Plugin/tests/bootstrap.php.twig [hunk 4]
```diff
@@ -55,3 +55,18 @@ if (file_exists($root . '/config/bootstrap.php')) {
...
+ */
+use Cake\TestSuite\Schema\SchemaManager;
+
+// Load a schema dump file.
+SchemaManager::create('test', './tests/schema.sql');
```
Comment (othercorey) [inline templates/bake/Plugin/tests/bootstrap.php.twig:72]:
```suggestion
SchemaManager::create('test', 'tests/schema.sql');
```

**Pull request merged**